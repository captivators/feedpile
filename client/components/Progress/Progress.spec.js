import React from 'react';
import { shallow, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Progress from './Progress'

test('Progress take a snapshot', () => {
  const component = shallow(<Progress />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});
