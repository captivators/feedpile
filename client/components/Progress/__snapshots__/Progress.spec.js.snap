// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress take a snapshot 1`] = `
<div
  className="progress"
>
  <CircularProgress
    color="#03A9F4"
    max={100}
    min={0}
    mode="indeterminate"
    size={60}
    thickness={7}
    value={0}
  />
</div>
`;
